python/mxnet/numpy/linalg.py

```diff
@@ -67,6 +70,53 @@ def matrix_rank(M, tol=None, hermitian=False):
     return _mx_nd_np.linalg.matrix_rank(M, tol, hermitian)
 
 
+def vecdot(a: ndarray, b, ndarray, axis: Optional[int] =None) -> ndarray:
```

Comment:
How about maintaining consistency in this file? the function above should take and return format inexplicitly; without `-> ndarray` annotation. If you want to keep the standard like this: then it could be nice to have all functions changed.